scripts/coverage/coverage.js

```diff
@@ -137,15 +138,16 @@ function reportCoverage () {
   const folders = {}
   for (const target of report.targets) {
     for (const file of target.files) {
-      if (!ignoreExps.some(exp => exp.test(file.path))) {
+      const content = readFileSync(file.path, 'utf8')
+      if (!ignoreExps.some(exp => exp.test(file.path)) && !ignoreContent.some(needle => content.includes(needle))) {
```

Comment:
Pretty cool!

Reply:
I was worried about loading up the content of each file, but we already were doing it later.